src/hotspot/share/opto/predicates.hpp

```diff
@@ -1176,6 +1176,7 @@ class CloneUnswitchedLoopPredicatesVisitor : public PredicateVisitor {
   ClonePredicateToTargetLoop _clone_predicate_to_false_path_loop;
 
   PhaseIdealLoop* const _phase;
+  bool const _is_counted_loop;
```

Comment:
We usually have them swapped for constants:
```suggestion
  const bool _is_counted_loop;
```